rcsb/db/wf/RepoLoadWorkflow.py

```diff
@@ -195,3 +199,39 @@ def buildResourceCache(self, rebuildCache=False):
         except Exception as e:
             logger.exception("Failing with %s", str(e))
         return ret
+
+    def removeAndRecreateDbCollections(self, op, **kwargs):
+        if op not in ["pdbx-db-wiper"]:
+            logger.error("Unsupported operation %r - exiting", op)
+            return False
+        try:
+            schemaLevel = kwargs.get("schemaLevel", "min") if kwargs.get("schemaLevel") in ["min", "full"] else "min"
+            dbType = kwargs.get("dbType", "mongo")
+            #
+            databaseName = kwargs.get("databaseName", None)
+            databaseNameList = self.__cfgOb.get("DATABASE_NAMES_ALL", sectionName="database_catalog_configuration").split(",")
+            collectionNameList = kwargs.get("collectionNameList", None)
+            #
+        except Exception as e:
+            logger.exception("Argument and configuration processing failing with %s", str(e))
+            return False
+        #
+        if op == "pdbx-db-wiper" and dbType == "mongo" and databaseName in databaseNameList:
```

Comment:
The first check seems unnecessary since it would have already bailed at the first line. But if there might be other supported ops in the future in this method, could be worth keeping.

Reply:
Good point. I was just copying the template from another method, but definitely this could be simplified. Thanks for pointing out!